common/common.scss

```diff
@@ -181,6 +181,18 @@ label.btn i {
   }
 }
 
+// Edge case for like button (double button)
+
+.double-button > .btn-text {
+    border-top-right-radius: 0px !important;
+    border-bottom-right-radius: 0px !important;
+}
+
+.double-button > .like-count + .btn-icon {
```

Comment:
This is to target the heart icon only when there is the like-count present. 
Otherwise, it will also incorrectly remove the border radius on posts with no likes